### ISSUE TITLE
Cobbler check for CentOS 7

### ISSUE THREAD

**Issue by vikas027** [cobbler/cobbler#1543]
I am on CentOS 7.1 x86_64 with cobbler-2.6.9-1.el7. Below command still tries to look for file  `/etc/xinetd.d/rsync`. Rsync is not managed by this file in CentOS 7

```bash
#  cobbler check | grep rsync
2 : file /etc/xinetd.d/rsync does not exist
#
# rpm -qa | grep ^cobbler
cobbler-2.6.9-1.el7.noarch
#
```

**Comment by alanoe** [comment]
If CentOS 7 uses xinet daemon, where are its configuration files?

**Comment by vikas027** [comment]
@alanoe  rsync is not managed by xinetd in CentOS 7, so `cobbler check` shouldn't look for that file. 

```bash
# rpm -ql rsync
/etc/rsyncd.conf
/etc/sysconfig/rsyncd
/usr/bin/rsync
/usr/lib/systemd/system/rsyncd.service
/usr/lib/systemd/system/rsyncd.socket
/usr/share/doc/rsync-3.0.9
/usr/share/doc/rsync-3.0.9/COPYING
/usr/share/doc/rsync-3.0.9/NEWS
/usr/share/doc/rsync-3.0.9/OLDNEWS
/usr/share/doc/rsync-3.0.9/README
/usr/share/doc/rsync-3.0.9/support
/usr/share/doc/rsync-3.0.9/support/Makefile
/usr/share/doc/rsync-3.0.9/support/atomic-rsync
/usr/share/doc/rsync-3.0.9/support/cvs2includes
/usr/share/doc/rsync-3.0.9/support/deny-rsync
/usr/share/doc/rsync-3.0.9/support/file-attr-restore
/usr/share/doc/rsync-3.0.9/support/files-to-excludes
/usr/share/doc/rsync-3.0.9/support/git-set-file-times
/usr/share/doc/rsync-3.0.9/support/logfilter
/usr/share/doc/rsync-3.0.9/support/lsh
/usr/share/doc/rsync-3.0.9/support/mnt-excl
/usr/share/doc/rsync-3.0.9/support/munge-symlinks
/usr/share/doc/rsync-3.0.9/support/rrsync
/usr/share/doc/rsync-3.0.9/support/rsyncstats
/usr/share/doc/rsync-3.0.9/support/savetransfer.c
/usr/share/doc/rsync-3.0.9/tech_report.tex
/usr/share/man/man1/rsync.1.gz
/usr/share/man/man5/rsyncd.conf.5.gz
```

**Comment by alanoe** [comment]
Clearer now, thanks

**Comment by jmaas** [comment]
Thanks for reporting this fix will be in the next release 2.6.11
